index.bs

```diff
@@ -123,20 +123,16 @@ Using Constructed Stylesheets {#using-constructed-stylesheets}
 =============================
 
 A {{CSSStyleSheet}} can only be applied to the {{Document}} it is constructed on (e.g. the document on which the factory function is called on),
-and any {{ShadowRoot}} in the {{Document}} it is constructed on, by adding a {{StyleSheetList}} containing the sheet to their {{adoptedStyleSheets}}.
+and any {{ShadowRoot}} in the {{Document}} it is constructed on, by adding an array containing the sheet to their {{adoptedStyleSheets}}.
```

Comment:
```suggestion
and any {{ShadowRoot}} in the {{Document}} it is constructed on, by assigning an array containing the sheet to their {{adoptedStyleSheets}}.
```

(this is a preexisting problem but nice to fix while we're here)

Reply:
Oh, thanks!